Cannot find module '../module/data/service/Notespace/NoteManager'

Error: src/lib/interval.ts(3,29): error TS2307: Cannot find module '../module/data/service/Notespace/NoteManager' or its corresponding type declarations.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! picode@1.0.0 build: `npx tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the picode@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-10-11T08_48_59_773Z-debug.log

![image](https://user-images.githubusercontent.com/28240077/136764682-8c7d8add-f7f3-45be-88cb-0d5241d1793f.png)

--- a/src/lib/interval.ts
+++ b/src/lib/interval.ts
@@ -1,6 +1,6 @@
 import DataChatManager from "../module/data/service/chatspace/chatManager";
 import DataCodeManager from "../module/data/service/codespace/codeManager";
-import DataNoteManager from "../module/data/service/Notespace/NoteManager";
+import DataNoteManager from "../module/data/service/notespace/noteManager";
 import DataDockerManager from "../module/data/service/workspace/dockerManager";
 
 export default class IntervalManager {

--- a/src/module/data/service/workspace/dockerManager.ts
+++ b/src/module/data/service/workspace/dockerManager.ts
@@ -24,6 +24,7 @@ import { ResponseCode } from "../../../../constants/response";
 
 const dockerNetworkFileName = "dockerNetworkList.json";
 const dockerInfoFileName = "dockerInfo.json";
+const defaultNetwork = ["host", "bridge", "none"];
 
 export default class DataDockerManager {
     static getDockerNetworkPath() {
@@ -105,7 +106,7 @@ export default class DataDockerManager {
         if (!isExists(this.getDockerNetworkPath())) {
             fs.mkdirSync(this.getDockerNetworkPath(), { recursive: true });
         }
-        ["host", "bridge", "none"].map((networkName) => {
+        defaultNetwork.map((networkName) => {
             const command = `docker network inspect --format="{{".IPAM.Config"}}" ${networkName}`;
             this.runDockerCommand(
                 command,
@@ -499,6 +500,14 @@ CMD ["./server-linux", "${socketPort}"]`;
         const updateOption = { disconnect: false, connect: false, link: false, rename: false };
         const newDockerInfo = this.getDockerInfo(workspaceId) as TDockerData;
         if (newDockerInfo === undefined) {
+            return false;
+        }
+
+        const networkInfo = this.getDockerNetworkInfo();
+        const defaultNetworkIdList = defaultNetwork.map((networkName: string) => {
+            return networkInfo[networkName].networkId;
+        });
+        if (defaultNetworkIdList.includes(dockerInfo.bridgeId)) {
             return false;
         }
 

